src/components/Lightbox/index.tsx

```diff
@@ -160,6 +185,14 @@ function Lightbox({
         }
     }, [hasSiblingCarouselItems, isActive, isFallbackVisible, isLightboxImageLoaded, isLightboxVisible]);
 
+    const handleScaleChange = useCallback(
```

Comment:
Callbacks should be named for what they do, not what they handle. Can you please change this name?